### PR TITLE
Cloning

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/containers/Auth/ConfirmEmailPage.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function ConfirmEmailPage(props) {
     confirmEmail(query).catch(() => {
       history.push('/login');
     });
-  }, [confirmEmail, history]);
+  }, [confirmEmail, history, query]);
```
Comment (simonebelotti75) [inline src/containers/Auth/ConfirmEmailPage.js:32]: Did you add it again?
I think I mentioned it before
As I remember, when we add this, it sends confirmation request multiple times

File: src/containers/Search/CloneModel.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  const dispatch = useDispatch();
+  const allProjects = useSelector((state) => state);
+  useEffect(() => {
+    dispatch(
+      loadMyProjectsAction(),
+    );
+  }, [dispatch]);
+
+  useEffect(() => {
+    allProjects.project.projects && setAllproject(allProjects.project.projects);
```
Comment (simonebelotti75) [inline src/containers/Search/CloneModel.js:26]: Please do not use this kind of approach
Use if condition

File: src/containers/Search/CloneModel.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                        // setactiveactivity();
+                      }
+                      // setActivePlaylist();
+                    }}
+                  >
+                    <div className="flex-bar">
+                      <div>
+                        {activeProject === counter_top + 1 ? (
+
+                          <FontAwesomeIcon icon="check-square" />
```
Comment (simonebelotti75) [inline src/containers/Search/CloneModel.js:58]: class mr-2 need

File: src/containers/Search/CloneModel.js [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+                        ) : (
+                            <FontAwesomeIcon icon="square" />
```
Comment (simonebelotti75) [inline src/containers/Search/CloneModel.js:60]: class mr-2 need

File: src/containers/Search/CloneModel.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+                                    <div>
+                                      <span>
+
+                                          {activePlaylist
+                                          === counter_playlist + counter_top + 1 ? (
+
+                                            <FontAwesomeIcon icon="stop-circle" />
+                                            ) : (
+                                              <FontAwesomeIcon icon="circle" />
+                                            )}
```
Comment (simonebelotti75) [inline src/containers/Search/CloneModel.js:109]: Even though we are running out of time, can you improve the code quality?
I am sure that you will not need more time to do that

File: src/containers/Search/style.scss [hunk 6]
```diff
@@ -269,9 +269,135 @@
...
+    span {
+      color: #Fff;
+      font-weight: 600;
+
+      svg {
+        margin-right:10px
+      }
+    }
+
+    ;
```
Comment (simonebelotti75) [inline src/containers/Search/style.scss:352]: Remove this

File: src/containers/Search/style.scss [hunk 7]
```diff
@@ -269,9 +269,135 @@
...
+    margin: 0 auto;
+    margin-top: 30px;
+    background-color: #fff;
+
+    &:hover {
+
+      color: #084892
+    }
+
+    ;
```
Comment (simonebelotti75) [inline src/containers/Search/style.scss:400]: Remove this

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]
Please improve your code quality, not related performance but just style for now
You will not need more time to do that
It's just habit

File: src/containers/Auth/ConfirmEmailPage.js (hunk 1 above)
Comment (muhammadqamar) [reply src/containers/Auth/ConfirmEmailPage.js]: I had removed that.
but i think eslint run command add it back.
can u verify again?

File: src/containers/Search/CloneModel.js (hunk 2 above)
Comment (muhammadqamar) [reply src/containers/Search/CloneModel.js]: done

File: src/containers/Search/CloneModel.js (hunk 3 above)
Comment (muhammadqamar) [reply src/containers/Search/CloneModel.js:60]: no need here. it is being cared through css already

File: src/containers/Search/CloneModel.js (hunk 4 above)
Comment (muhammadqamar) [reply src/containers/Search/CloneModel.js:62]: no need here. it is being cared through css already

File: src/containers/Search/CloneModel.js (hunk 5 above)
Comment (muhammadqamar) [reply src/containers/Search/CloneModel.js]: done

File: src/containers/Search/style.scss (hunk 6 above)
Comment (muhammadqamar) [reply src/containers/Search/style.scss]: done

File: src/containers/Search/style.scss (hunk 7 above)
Comment (muhammadqamar) [reply src/containers/Search/style.scss]: done

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**